### PR TITLE
update angular to version 12

### PR DESCRIPTION
**Pull request by simonmartinez**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report package#1166.1 ✘  
* Stage 1
  * Docker build ✘

**Comment by ovh-cds** [comment]
CDS Report package#1167.0 ✘  
* Stage 1
  * Docker build ✘

**Comment by ovh-cds** [comment]
CDS Report package#1168.0 ✘  
* Stage 1
  * Docker build ✘

**Comment by ovh-cds** [comment]
CDS Report package#1169.0 ✘  
* Stage 1
  * Docker build ✘

**Comment by ovh-cds** [comment]
CDS Report package#1170.0 ✘  
* Stage 1
  * Docker build ✘

**Comment by ovh-cds** [comment]
CDS Report package#1172.0 ✘  
* Stage 1
  * Docker build ✘

**Comment by ovh-cds** [comment]
CDS Report package#1173.0 ✘  
* Stage 1
  * Docker build ✘

File: ui/dashboard/package.json [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
+    "@angular/forms": "12.2.12",
+    "@angular/localize": "12.2.12",
+    "@angular/platform-browser": "12.2.12",
+    "@angular/platform-browser-dynamic": "12.2.12",
+    "@angular/router": "12.2.12",
+    "@sentry/browser": "5.27.4",
+    "@swimlane/ngx-charts": "^19.1.0",
+    "@types/lodash-es": "4.17.3",
+    "convert-yaml": "2.0.5",
+    "d3-drag": "^3.0.0",
```
Comment (richardlt) [inline ui/dashboard/package.json:30]: Can you set the version (remove the ^) for this one and others ?

**Review by richardlt — APPROVED** [review]

**Review by rbeuque74 — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Pull request merged**